### ISSUE TITLE
Update Zooz firmware

### ISSUE THREAD

**Issue by rlippmann** [zwave-js/firmware-updates#145]
It appears that Zooz updated a bunch of firmware files recently, particularly on the ZEN-7x series.

It would be nice to be able to have zwave-js update them.

**Comment by AlCalzone** [comment]
Yeah, I merged some updates from Zooz yesterday: https://github.com/zwave-js/firmware-updates/pull/149

> My only complaint would be the ui method didn't have a nice progress bar like HA did.

There should be a progress indicator next to the device in the device list.
